src/SonarQube.Client/Services/ISonarQubeService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using SonarQube.Client.Models;
+
+namespace SonarQube.Client.Services
+{
+    public interface ISonarQubeService
```

Comment:
What's our normal practice about documenting interfaces?

Reply:
Normal practice is no documentation at all (not only interface). I don't mind either way (although I am a bit reluctant to too much doc cause it always end up in dumb - not helpeful - comments).